app/app.go

```diff
@@ -175,11 +176,13 @@ var (
 		staking.AppModuleBasic{},
 		distr.AppModuleBasic{},
 		gov.NewAppModuleBasic(
-			paramsclient.ProposalHandler, distrclient.ProposalHandler, upgradeclient.ProposalHandler, upgradeclient.CancelProposalHandler,
-			ibcclientclient.UpdateClientProposalHandler, ibcclientclient.UpgradeProposalHandler,
-			// Evmos proposal types
-			erc20client.RegisterCoinProposalHandler, erc20client.RegisterERC20ProposalHandler, erc20client.ToggleTokenConversionProposalHandler,
-			incentivesclient.RegisterIncentiveProposalHandler, incentivesclient.CancelIncentiveProposalHandler,
+			[]govclient.ProposalHandler{
```

Comment:
most of the time, I will follow practices from the SDK's simapp because it is basically the writer's recommendation. So, I see that it is still kept in simapp/app.go.

https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/simapp/app.go#L303